_pages/integrations/arize.md

```diff
@@ -122,7 +123,8 @@ shap_values = get_shap_values(clf, X, show_graph=True)
 
 <img src="{{site.cdnurl}}{{site.baseurl}}/images/post_images/arize/shap_values.png" alt="Shap values plot in Jupyter notebook" class="screenshot">
 
-The following represents the algorithm code that you would deploy on Algorithma, not the training platform used above. {:.notice-info}
+The following represents the algorithm code that you would deploy on Algorithma, not the training platform used above.
+{:.notice-info}
```

Comment:
Wow that's all it was? Good to know for future reviews.

Reply:
well..do we know that worked??